modules/mongodb/manifests/s3backup/package.pp

```diff
@@ -1,9 +1,9 @@
 #
 class mongodb::s3backup::package {
 
-  $packages = ['boto3', 'python-dateutil', 'simplejson']
+  
 
-  package { $packages:
+  package { 's3cmd' :
```

Comment:
If we're using s3cmd we probably need to add an `.s3cfg` file otherwise it complains (might create a module for this package at some point)